libs/components/modals/src/lib/modules/modal/modal.component.spec.ts

```diff
@@ -80,24 +69,10 @@ describe('Modal component', () => {
       imports: [SkyModalFixturesModule],
     });
 
-    testModals = [];
+    // Confirm all modals are closed before another test is executed.
+    expect(SkyModalHostService.openModalCount).toBe(0);
```

Comment:
Capturing "test modals" is no longer required; simply checking the open modal count is sufficient.